lib/field.ts

```diff
@@ -60,11 +60,18 @@ export default class Field {
     return this._getFieldLocale(locale).onIsDisabledChanged(handler)
   }
 
-  private _getFieldLocale(locale) {
-    locale = locale || this._defaultLocale
```

Comment:
I think we should completely revert this change back to the way it was as the new logic is actually functionally different.

In this case if `locale` was falsy i.e. `''`, `0`,`undefined`, `null`, it would use the default.

In the new logic, using a default parameter only checks for `undefined`. Every other falsy value will NOT use the default.

The only caveat to this comment is if `assertHasLocale` would have done that for us. Either way it's probably safer to just not touch any of this code now that we are just introducing a new method completely.